hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3ARemoteFileChanged.java

```diff
@@ -326,7 +326,7 @@ protected Path path() throws IOException {
    * @return a number >= 0.
    */
   private int getFileStatusHeadCount() {
-    return authMode ? 0 : 0;
```

Comment:
interesting, why it was 0 : 0? I really haven't noticed it last time

Reply:
it was 1: 0 and th

Reply:
it was originally authMode ? 0 : 1; I'd switched it to 0 : 0 as things changed -but didn't actually deleting, so am reinstating the old code